### ISSUE TITLE
Possible to keep date when task moved to Done?

### ISSUE THREAD

**Issue by grepsuzette** [smallhadroncollider/taskell#79]
I'd like to keep the date when a card is moved to the Done column. 
Both for timeframe and even more to help prepare reports.
Once they all disappeared entries in the done column become more useless for me.

What is the rationale for them to disappear?
1. It must be because it appears in green normally and in red after the date? But for me all red as long as it's in a Done column is ok.
2. It must be also for saving 1 line of text per cards in Done, for people or devs who do many cards a day :)   

Maybe an option in the config file?

**Comment by smallhadroncollider** [comment]
The dates were designed as due dates/deadlines. In that thinking once the task is complete, the due date is no longer relevant. I don't want to see tasks that have due dates but that I've completed.

Sounds like you're using them for a different purpose. I'll have a think about adding this as an option. For now you could use `L` to move cards to the right or use `m` to move tasks to the the "Done" list.

**Comment by emugel** [comment]
Ah noted! Thank you :)
